tools/build/configure.py

```diff
@@ -941,7 +941,7 @@ def build(
                     entry.src_paths,
                     "icons",
                     variables={
-                        "list_path": entry.src_paths[0],
+                        "list_path": self.resolve_asset_path(entry.src_paths[0]),
```

Comment:
We pass the unresolved path to the script because it's supposed to resolve it. So if it's not doing that, I think it's a bug in the build script

Reply:
Ah, that makes sense. After spending so much time tracking this one down I didn't consider that!
Perhaps unresolved asset paths should be `assets/__not_resolved__/...` instead of defaulting to the final path in the stack, so that we can catch bugs like this in future

Reply:
I guesh we could do that, but it's the path of least resistance to not mutate the path before giving it to that script

Reply:
I think there's a misunderstanding here. I'll update the script to do the resolving